diff --git a/CMakeLists.txt b/CMakeLists.txt
index 5cc2d18..e6f98cc 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -226,13 +226,11 @@ if(UNIX AND NOT APPLE)
 	if(OPENSPADES_RESOURCES)
 		# install asset paks (including non-GPL one)
 		install(FILES
-			${CMAKE_BINARY_DIR}/Resources/pak000-Nonfree.pak
 			${CMAKE_BINARY_DIR}/Resources/pak002-Base.pak
 			${CMAKE_BINARY_DIR}/Resources/pak005-Models.pak
 			${CMAKE_BINARY_DIR}/Resources/pak010-BaseSkin.pak
 			${CMAKE_BINARY_DIR}/Resources/pak050-Locales.pak
 			${CMAKE_BINARY_DIR}/Resources/pak999-References.pak
-			${CMAKE_BINARY_DIR}/Resources/font-unifont.pak
 			DESTINATION ${OPENSPADES_INSTALL_RESOURCES})
 	endif(OPENSPADES_RESOURCES)
 
diff --git a/Resources/CMakeLists.txt b/Resources/CMakeLists.txt
index 1f9b514..34010fa 100644
--- a/Resources/CMakeLists.txt
+++ b/Resources/CMakeLists.txt
@@ -4,11 +4,6 @@ if(OPENSPADES_RESOURCES)
 	if (WIN32)
 		# No automatic downloading for Windows (for now)
 	elseif (UNIX)
-		add_custom_target(OpenSpades_Resources_DevPaks ALL COMMENT "Downloading non-GPL assets")
-		add_custom_command(
-			TARGET OpenSpades_Resources_DevPaks
-			COMMAND ${CMAKE_CURRENT_SOURCE_DIR}/downloadpak.sh
-			WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR})
 	endif()
 
# TODO: subgroups for script files