diff --git a/Resources/Icons/converticons.sh b/Resources/Icons/converticons.sh
index 1e8dc9b..46d504f 100755
--- a/Resources/Icons/converticons.sh
+++ b/Resources/Icons/converticons.sh
@@ -21,7 +21,7 @@ rm tmp/40x40.png # nobody uses icons with such res
 for fn in tmp/*.png; do
 	RES=$( basename $fn | sed 's/.png//' )
 	mkdir -p "$OUTPUT_DIR/$RES/apps"
-	mv $fn "$OUTPUT_DIR/$RES/apps/openspades.png"
+	mv $fn "$OUTPUT_DIR/$RES/apps/jp.yvt.OpenSpades.png"
 done
 
 rm -rf tmp/
diff --git a/Resources/Unix/Desktop/openspades.desktop b/Resources/Unix/Desktop/openspades.desktop
index 985d474..2a7809c 100644
--- a/Resources/Unix/Desktop/openspades.desktop
+++ b/Resources/Unix/Desktop/openspades.desktop
@@ -7,6 +7,6 @@ Name=OpenSpades
 GenericName=Sandbox building and FPS videogame
 Comment=Open-source clone of Ace of Spades
 Keywords=game;fps;sandbox;voxel;aos;
-Icon=openspades
+Icon=jp.yvt.OpenSpades
 Categories=Game;ActionGame;
MimeType=x-scheme-handler/aos;